tools/convert_to_16p2

```diff
@@ -66,19 +63,27 @@ ie. 16-bit yuv planar data samples with the active bits in the high order bits
 
     args = parser.parse_args()
 
-    f = open(args.infilename, "rb")
-
     outfilename = args.infilename + ".16p2"
+    of = open(outfilename, "wb")
 
-    data = bytearray(f.read())
+    word = bytearray(2)
+    temp = bytearray(1)
```

Comment:
This bytearray is later overwritten by an integer (e.g. `temp = word[1]` gets the value of byte 1 of `word` as an int and assigns to to the variable `temp`, i.e. it doesn't put byte 1 in the byte array you created here).

Delete this line?

Reply:
Done in aed6f5d